examples/pojo-jar/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Compiled class file
```

Comment:
Is that file really needed? There already is a top-level `.gitignore` file in this repository.

Reply:
Will remove it